Wrapping/Python/Python.i

```diff
@@ -19,6 +19,10 @@
 
 %begin %{
 #include "sitkPython.h"
+// Needed by SWIG, but not in Stable Python API.
+// https://github.com/swig/swig/pull/2856
+#define NOMINMAX
```

Comment:
@thewtex I had the weird error messages caused by MSVC headers defining min and max. I saw your PR for SWIG do you think this should go into SWIG too?

Reply:
@blowekamp I recall something like this a long time ago, but I have not seen it since then.